builders/macos-python-builder.psm1

```diff
@@ -22,6 +22,22 @@ class macOSPythonBuilder : NixPythonBuilder {
         [string] $platform
     ) : Base($version, $architecture, $platform) { }
 
+    [void] PrepareEnvironment() {
+        <#
+        .SYNOPSIS
+        Prepare system environment by installing dependencies and required packages.
+        #>
+
+        ### zlib is needed on the testing stage
+        if ($this.Version -lt "3.7.0") {
+            Execute-Command -Command "brew install zlib"
```

Comment:
will it work on the final image as it doesn't have zlib?

Reply:
@miketimofeev given that our python is dynamically linked it will work as long as someone is not running the code that requires zlib. zlib itself is widely used so it makes sense to install it during images generation.

Reply:
zlib is in the image now awaiting deployment.